tfjs-core/src/hash_util.ts

```diff
@@ -14,8 +14,11 @@
  * limitations under the License.
  * =============================================================================
  */
-// tslint:disable-next-line:no-require-imports variable-name
-const Long = require('long');
+// Workaround for allowing cjs module to be included in bundle created by
+// rollup.
+import * as LongExports from 'long';
+// tslint:disable-next-line
+const Long = (LongExports as any).default || LongExports;
```

Comment:
IIRC, this is what 'esModuleInterop' does, so this approach looks good given the issues we've had with enabling 'esModuleInterop'.

Is the type of `Long` `any` now? We might want to cast it to `typeof Long`.